src/current/v25.2/upgrade-cockroachdb-kubernetes-operator.md

```diff
@@ -0,0 +1,139 @@
+    ```
+
+    The Operator will perform the staged update.
+
+3. To check the status of the rolling upgrade, run `kubectl get pods`.
+
+4. Verify that all pods have been upgraded:
+
+    ```shell
+    $ kubectl get pods \
+    -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.containers[0].image}{"\n"}'
+    ```
+
+You can also check the CockroachDB version of each node in the [DB Console](ui-cluster-overview-page.html#node-details).
+
+### Roll back a patch upgrade
+
+{% include_cached common/upgrade/patch-rollback-kubernetes.md %}
+
+## Perform a major-version upgrade
+
+To perform a major upgrade:
+
+1. Change the container image in the values file:
+
+    ```yaml
+    cockroachdb:
+      crdbCluster:
+        image:
+          name: cockroachdb/cockroach:v25.1.4
+    ```
+
+2. Apply the new settings to the cluster:
+
+    ```shell
+    $ helm upgrade --reuse-values $CRDBCLUSTER ./cockroachdb-parent/charts/cockroachdb --values ./cockroachdb-parent/charts/cockroachdb/values.yaml -n $NAMESPACE
+    ```
+
+    The Operator will perform the staged update.
+
+3. To check the status of the rolling upgrade, run `kubectl get pods`.
+
+4. Verify that all pods have been upgraded:
+
+    ```shell
+    $ kubectl get pods \
+    -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.containers[0].image}{"\n"}'
+    ```
+
+5. If auto-finalization is enabled (the default), finalization begins as soon as the last node rejoins the cluster. When finalization finishes, the upgrade is complete.
+
+6. If auto-finalization is disabled, follow your organization's testing procedures to decide whether to [finalize the upgrade](#finalize-a-major-version-upgrade-manually) or [roll back](#roll-back-a-major-version-upgrade) the upgrade. After finalization begins, you can no longer roll back to the cluster's previous major version.
+
+### Finalize a major-version upgrade manually
+
+{% include common/upgrade/finalize-kubernetes.md %}
+
+### Roll back a major-version upgrade
+
+To roll back to the previous major version before an upgrade is finalized:
+
+1. Change the container image in the custom resource to use the previous major version:
+
+    ```yaml
+    cockroachdb:
+      crdbCluster:
+        image:
+          name: cockroachdb/cockroach:v24.3
+    ```
+
+2. Apply the new settings to the cluster:
+
+    ```shell
+    $ helm upgrade --reuse-values $CRDBCLUSTER ./cockroachdb-parent/charts/cockroachdb --values ./cockroachdb-parent/charts/cockroachdb/values.yaml -n $NAMESPACE
+    ```
+
+    The Operator will perform the staged rollback.
+
+3. To check the status of the rollback, run `kubectl get pods`.
+
+4. Verify that all pods have been rolled back:
+
+    ```shell
+    $ kubectl get pods \
+    -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.containers[0].image}{"\n"}'
+    ```
+
+Rollbacks do not require finalization.
+
+## Disable auto-finalization
+
+{% include common/upgrade/disable-auto-finalization.md %}
```

Comment:
This is not rendering like it should, unsure why. The needed version of the include content can be found here: https://docs.google.com/document/d/196HI1b1QSbyDGtLPT3FF8rbNeMVdOZBCMyQ7UKdbK8k/edit?tab=t.k5y17crpdkaz#heading=h.144douprth3q

@mikeCRL can you take a look?

Reply:
Fixed - was an issue with the liquid conditionals, since the operator doc had `kubernetes` in the full filename but didn't use the Operator/Manual/Helm selectors so the matched content had nothing that would render on the site. Solution was to switch from using `path contains` as the conditional to the more specific `file.name ==`